rfcs/2022-04-09-accessibility.md

```diff
@@ -0,0 +1,495 @@
+---
+Start Date: 08-04-2022
+RFC PR: (leave this empty if no PR yet)
+Blade Issue: (leave this empty if no issue yet)
+---
+
+# Blade Accessibility <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+
+
+## What is Accessibility?
+
+Accessibility is the practice of making websites usable by people of all abilities and disabilities. This includes people with visual, auditory, physical, and cognitive impairments, as well as those with temporary disabilities such as broken bones, color blindness, and temporary loss of vision, but the practice of making sites accessible also benefits other groups such as people using mobile phones, people with temporary disabilities, situational limitations.
+
+## Accessibility for who? 
+
+Accessibility is essential for some, useful for all. It's not just for people with disabilities, accessible websites also improve the user experience of normal users. 
+
+For example:  
+
+- Prodiving captions for a video can benefit users who watch videos in loud environments or in a library. 
+- Sufficient color contrast is good for different lighting conditions and people with visual impairments.
+- Older people with changing abilities due to ageing can benefit from accessible experiences. 
+- Good layouts and design helps users with cognitive disabilities but also people who are unconfident with technology or old aged people.
+- Large buttons and links helps people with reduced dexterity but also improves user experience of mobile devices.
+- Ensuring keyboard accessibility helps people with temporary/permanent mobility but also improves experience for normal users by providing better user experience for components like Select or Modals. 
+
+> Check out this youtube [video about Web Accessibility Perspectives](https://www.youtube.com/watch?v=3f31oufqFSM) by W3C for more details about how accessibility can help wide range of people with or without disabilities.
+
+## Various types disabilities
+
+- **Visual:**  
+  Anything that deals with sight, eg: people who have different types of sight like color blindness.
+- **Cognitive:**  
+  Concentration, memory, judgment, problem solving, logic skills.
+- **Mobility:**  
+  Anything that affects movement in a body.
```

Comment:
Okay will rephrase this, what this means that say people who have mobility issues eg: broken/missing arm should also be able to operate our sites. 

This goes under having proper keyboard accessibility because people with mobility issues might not be as accurate with mouse.